### DISCUSSION TITLE
OpenSees driver

### DISCUSSION THREAD

**Discussion by childofhappiness21** [snl-dakota/dakota#23] [Q&A]
Hi all,

 I have an OpenSees model and want to do sensitivity analysis using Dakota. I am running it in Compute Canada cedar, Canadian high performance computing center and need with the following support:

 1. How to connect the OpenSees program with Dakota (especifically, on how to define/ create analysis_driver in Dakota that can be communicate with OpenSees.
 2. I got an example in this video: https://www.youtube.com/watch?v=ZeWNMJbbKBM (41:47 to 43:37). I used the same procedure but I am getting this message "sh: ./opensees_driver_1.sh: Permission denied. 
                                   Error: cannot open results file "results.out" for evaluation 1"

I look forward for your invaluable support. 

Thanks

**Comment by jadamstephens** [comment]
I'm not familiar with OpenSees, so I won't be able to give much specific help.

Almost always, the cause of a "Permission denied" error on a driver is that the driver doesn't have execute permission. Use this command to set it:

`chmod +x opensees_driver_1.sh`

You may find our [interfacing examples](https://github.com/snl-dakota/dakota-examples/tree/master/official/drivers) and our documentation for [coupling Dakota to a simulation](https://dakota.sandia.gov/sites/default/files/docs/6.17.0-release/user-html/usingdakota/introduction/couplingtosimulations.html) helpful.

> your invaluable support

I assure you that my support is not invaluable. I get paid to do this. ;)

**Reply by childofhappiness21** [reply]
Thanks, I will go through them.
